### PR TITLE
The redis config file must be the first parameter

### PR DESCRIPTION
**Pull request by bbkz**
otherwise the redis-server binary will crash with

> Bad directive or wrong number of arguments

### CODE REVIEW

File: charts/redis/templates/statefulset.yaml [hunk 1]
```diff
@@ -123,10 +123,10 @@ spec:
           command:
             - redis-server
           args:
+            - /data/conf/redis.conf
```
Comment (groundhog2k) [inline charts/redis/templates/statefulset.yaml:126]: Please apply the same fix for the redis-sentinel container (around line 195)

**Review by groundhog2k — CHANGES_REQUESTED** [review]
Thanks for finding and fixing this! 
Please apply the fix also to the sentinel-container.

Update the Chart version in Chart.yaml and README to 0.4.6.
(Thank you!)

**Comment by groundhog2k** [comment]
Closes #456

**Comment by bbkz** [comment]
Hi,

i added the change for sentinel and updated the chart version.

Yours,
bbk

**Review by groundhog2k — APPROVED** [review]

**Pull request merged**